.rubocop.yml

```diff
@@ -98,6 +98,14 @@ RSpec/PredicateMatcher:
 RSpec/DescribeClass:
   Enabled: false
 
+RSpec/VariableDefinition:
+  Exclude:
+    - spec/mailers/log_email_delivery_observer_spec.rb
+
+RSpec/VariableName:
+  Exclude:
+    - spec/mailers/log_email_delivery_observer_spec.rb
```

Comment:
rubocop seems to treat `subject` as an rspec keyword instead of `subject` as an attribute/method on an email